packages/@pollyjs/adapter-node-http/src/index.js

```diff
@@ -0,0 +1,69 @@
+import Adapter from '@pollyjs/adapter';
+
+import HttpWrapper from './utils/http-wrapper';
+
+export default class HttpAdapter extends Adapter {
+  static get name() {
+    return 'node-http';
+  }
+
+  get defaultOptions() {
+    return {
+      transports: {
```

Comment:
For consistency with the other adapters, I'd prefer this to be `['http', 'https']`.

Reply:
Done in 25589639be340c56e0fcd56d85d13c36585fde2f